vspec/vssexporters/vss2jsonschema.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+
+# (c) 2023 BMW Group
+#
+# All files and artifacts in this repository are licensed under the
+# provisions of the license provided by the LICENSE file in this repository.
+#
+#
+# Convert vspec tree to JSON Schema
+
+from vspec.model.vsstree import VSSNode
+import argparse
+import json
+import logging
+from typing import Dict, Any
+from vspec.loggingconfig import initLogging
+
+type_map = {
+    "int8": "integer",
+    "uint8": "integer",
+    "int16": "integer",
+    "uint16": "integer",
+    "int32": "integer",
+    "uint32": "integer",
+    "int64": "integer",
+    "uint64": "integer",
+    "boolean": "boolean",
+    "float": "number",
+    "double": "number",
+    "string": "string",
+    "int8[]": "array",
+    "uint8[]": "array",
+    "int16[]": "array",
+    "uint16[]": "array",
+    "int32[]": "array",
+    "uint32[]": "array",
+    "int64[]": "array",
+    "uint64[]": "array",
+    "boolean[]": "array",
+    "float[]": "array",
+    "double[]": "array",
+    "string[]": "array"
+}
+
+def add_arguments(parser: argparse.ArgumentParser):
+    parser.description = "The JSON schema exporter does not support any additional arguments."
```

Comment:
Should we possibly have `--json-pretty` similar to vspec2json?

Reply:
i guess it should work same as for JSON.